cmd/nerdctl/run_windows.go

```diff
@@ -35,6 +35,8 @@ const (
 
 	// HostProcessContainer will launch a host process container
 	hostProcessContainer = "microsoft.com/hostprocess-container"
+	UVMMemoryAnnotations = "MemorySizeInMB"
```

Comment:
```suggestion
	uvmMemorySizeInMB = "io.microsoft.virtualmachine.computetopology.memory.sizeinmb"
```

Reply:
I hate having to redefine these here, but otherwise we'd need to pull them from hcsshim, which means we need an hcsshim dep 😬. That'd feel like a waste also